ups/lsst_ci.table

```diff
@@ -0,0 +1,12 @@
+setupRequired(lsst)
+setupRequired(lsst_apps)
+setupRequired(lsst_distrib)
+
+setupRequired(testdata_subaru)
+setupRequired(testdata_cfht)
+setupRequired(testdata_decam)
+setupRequired(obs_subaru)
+setupRequired(obs_cfht)
+setupRequired(obs_decam)
+
+setupRequired(obs_lsstSim)
```

Comment:
You may want to explicitly include `obs_sdss` for completeness. I know it's part of `lsst_apps`.